sql_generators/funnels/configs/android_onboarding.toml

```
@@ -55,8 +55,8 @@ friendly_name = "First Card"
 description = "First Onboarding Card Impression"
 data_source = "onboarding_events"
 # join_previous_step_on = "ac.client_id"
-select_expression = """CASE WHEN `mozfun.map.get_key`(event_extra, 'sequence_position') = '1' AND
-    `mozfun.map.get_key`(event_extra, 'action') = 'impression' AND
+select_expression = """CASE WHEN JSON_EXTRACT_SCALAR(event_extra.sequence_position)  = '1' AND
```

`JSON_EXTRACT_SCALAR` is [deprecated](https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions#json_extract_scalar).  I think `JSON_VALUE` is equivalent
```suggestion
select_expression = """CASE WHEN JSON_VALUE(event_extra.sequence_position) = '1' AND
```

Also nit but there's an extra space added before the `=`